helpers/p2p_docs/doc_data/types/GrapheneIblt.txt

```diff
@@ -0,0 +1,12 @@
+
+\startformat
+hash table
+vector<HashTableEntry>
```

Comment:
I believe this was renamed to `IBLTEntry`. Did I forgot to rename somewhere?

Reply:
No, I think the code is fine. It doesn't have a direct representation in messages.py what I was using for some testing. There it's named GrapheneIbltEntryDummy. I will rename it to IBLTEntry in the docs.

Reply:
Ah, yes, in `messages.py` it was named `*Dummy` because it does not provide actual IBLT functionality. Only serialization/deserialization